servicetalk-tcp-netty-internal/src/main/java/io/servicetalk/tcp/netty/internal/TcpServerBinder.java

```diff
@@ -154,7 +156,7 @@ protected void handleSubscribe(Subscriber<? super ServerContext> subscriber) {
                     Throwable cause = f.cause();
                     if (cause == null) {
                         subscriber.onSuccess(NettyServerContext.wrap(channel, channelSet,
-                                connectionAcceptor, executionContext));
+                                connectionAcceptor, executionContext, config.isAsyncCloseOffloaded()));
```

Comment:
just thinking out loud to make sure I follow  ... having a single flag will make it so offloading is excessive for consumers of `ServerContext` (e.g. only need to offload the `ChannelSet`), however this approach is "safe" bcz a service could close the connection via the `HttpServiceContext`. If we wanted to we could add another flag in the future so users can offload just the `ChannelSet` in a non-breaking fashion.